internal/services/policy/remediation_management_group.go

```diff
@@ -95,6 +95,8 @@ func resourceArmManagementGroupPolicyRemediation() *pluginsdk.Resource {
 			"policy_definition_reference_id": {
 				Type:     pluginsdk.TypeString,
 				Optional: true,
+				// TODO: remove this suppression when github issue https://github.com/Azure/azure-rest-api-specs/issues/8353 is addressed
+				DiffSuppressFunc: suppress.CaseDifference,
```

Comment:
sounds like we're still waiting on the service team here - can we reach out to them here? https://github.com/Azure/azure-rest-api-specs/issues/8353#issuecomment-1020486348

Reply:
Thanks for your suggestion ❤️

Reply:
Blocked on the Service Team: https://github.com/Azure/azure-rest-api-specs/issues/8353#issuecomment-1298597542